docs/content/preview/explore/fault-tolerance/macos.md

```diff
@@ -17,74 +17,62 @@ menu:
 type: docs
 ---
 
-YugabyteDB can automatically handle failures and therefore provides [high availability](../../../architecture/core-functions/high-availability/). This tutorial demonstrates how YugabyteDB can continue to do reads and writes even in case of node failures. You create YSQL tables with a replication factor (RF) of 3, which allows a [fault tolerance](../../../architecture/docdb-replication/replication/) of 1. This means the cluster remains available for both reads and writes even if one node fails. However, if another node fails (bringing the number of failures to two), writes become unavailable on the cluster to preserve data consistency.
+YugabyteDB can automatically handle failures and therefore provides [high availability](../../../architecture/core-functions/high-availability/). The recovery point objective (RPO) for the tablets in a YugabyteDB cluster is 0, meaning no data is lost in the failover to another zone. The recovery time objective (RTO) is 3 seconds, which is the time window for completing the failover and becoming operational out of the remaining zones.
+
+This tutorial demonstrates how YugabyteDB can continue to do reads and writes even in case of node failures. You create YSQL tables with a replication factor (RF) of 3, which allows a [fault tolerance](../../../architecture/docdb-replication/replication/) of 1. This means the cluster remains available for both reads and writes even if a node or an entire zone or region fails. However, if another node were to fail (bringing the number of failures to two), writes become unavailable on the cluster to preserve data consistency.
```

Comment:
```suggestion
This tutorial demonstrates how YugabyteDB can continue to do reads and writes even in case of node failures. You create YSQL tables with a replication factor (RF) of 3, which allows a [fault tolerance](../../../architecture/docdb-replication/replication/#fault-tolerance/) of 1. This means the cluster remains available for both reads and writes even if a node or an entire zone or region fails. However, if another node were to fail (bringing the number of failures to two), writes become unavailable on the cluster to preserve data consistency.
```

Reply:
You are linking to the page at the end in Learn more anyway. So you might as well get more specific with the link here.